x-pack/solutions/observability/plugins/infra/public/alerting/log_threshold/components/expression_editor/editor.tsx

```diff
@@ -289,34 +298,41 @@ export const Editor: React.FC<RuleTypeParamsExpressionProps<PartialRuleParams, L
   return (
     <>
       {resolvedLogView && <LogViewSwitcher logView={resolvedLogView} />}
-
+      <EuiText size="xs">
+        {i18n.translate('xpack.infra.editor.modifyLogViewSetting', {
+          defaultMessage: 'To modify go to ',
+        })}
+        <EuiLink
+          data-test-subj="infraEditorLinkToAdvancedSettings"
+          href={http.basePath.prepend('/app/management/kibana/settings?query=Log+sources')}
```

Comment:
I think this link should be space aware as this setting can be different in different spaces

Reply:
basePath wouldn't take care of spaces?

Reply:
you are right, it does!